### PR TITLE
Test for egison command-line options

### PR DESCRIPTION
**Pull request by matsubara0507**
Resolve: #244

Implement test for `egison` command-line options. 
refs https://egison.readthedocs.io/en/latest/reference/command-line-options.html

```
$ stack test egison:test-cli
egison> test (suite: test-cli)

:load-file option: [OK]
:load-file option: [OK]
:eval option: [OK]
:command option: [OK]
:TSV option: [OK]
:TSV option with tab: [OK]
:substitute option: [OK]
:map option: [OK]
:filter option: [OK]
:field option: [OK]
:math option: [OK]
:sexpr option: [OK]

         Test Cases   Total       
 Passed  12           12          
 Failed  0            0           
 Total   12           12          

egison> Test suite test-cli passed
```

NOTE: This test is depend on stack command.

### CODE REVIEW

File: test/fixture/name-of-file-to-load.egi [hunk 1]
```diff
@@ -0,0 +1 @@
+def x := 1
```
Comment (matsubara0507) [inline test/fixture/name-of-file-to-load.egi:1]: https://github.com/egison/egison/blob/8ebeacaa37835f4ff5703f4277f25c9165001021/docs/reference/command-line-options.rst#L11-L12

write `x := 1` in document.
but, occur error

```
$ stack exec -- egison -l test/fixture/name-of-file-to-load.egi 
Parse error at: egison:1:3:
  |
1 | x := 1
  |   ^
unexpected ':'
expecting "...", "where", atomic expression, end of input, index, or infix
```

should I fix documents in this PR? 
or, different PR?

Comment (momohatt) [reply test/fixture/name-of-file-to-load.egi:1]: Right, we changed the syntax from v4.1.0 to require `def` keyword before global definitions, but we haven't updated the docs yet.
Feel free to open another PR to fix this (if you have time, of course)

File: test/OptionsTest.hs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import           Test.Framework.Providers.HUnit (hUnitTestToTests)
+import           Test.HUnit
+
+import           Language.Egison                (version)
+
+main :: IO ()
+main = defaultMain . hUnitTestToTests . test $ TestList
+    [ TestLabel "load-file option" . TestCase $ assertEgisonCmd
+        (interpreter "1\n")
+        ["--load-file", "test/fixture/name-of-file-to-load.egi"]
```
Comment (momohatt) [inline test/OptionsTest.hs:16]: In [this example in the docs](https://egison.readthedocs.io/en/latest/reference/command-line-options.html#l-load-file), the name `name-of-file-to-load.egi` and `name-of-file-to-test.egi` is meant to be replaced with some other names, and using these names in the tests feels a bit strange.
Can you rename it? (`a.egi`, for example?)

File: test/OptionsTest.hs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        "x"
+    , TestLabel "load-file option" . TestCase $ assertEgisonCmd
```
Comment (momohatt) [inline test/OptionsTest.hs:18]: `"load-file option"` -> `"test option"`?

**Review by momohatt — CHANGES_REQUESTED** [review]
Thanks for the great PR! Mostly looks good.

File: test/OptionsTest.hs (hunk 2 above)
Comment (matsubara0507) [reply test/OptionsTest.hs]: rename to `a.egi` `b.egi`

File: test/OptionsTest.hs (hunk 3 above)
Comment (matsubara0507) [reply test/OptionsTest.hs]: fixit

**Review by momohatt — APPROVED** [review]

**Comment by momohatt** [comment]
LGTM, thanks a lot!

**Pull request merged**